src/views/Home.vue

```diff
@@ -1,34 +1,32 @@
 <template>
-<div class="img" v-bind:style="{ backgroundImage: 'url(' + image + ')' }">
-    <my-banner />
-<b-container>
-<div class="row" style="padding-top: 5%;padding-bottom: 5%;">
-    <div class="col-md-4">
-    <my-grid/>
-    </div> 
-    <div class="col-md-4">
-    <my-grid/>	
-    </div> 
-    <div class="col-md-4">
-        <my-grid/>
-    </div> 
-</div> 
+    <div class="img" v-bind:style="{ backgroundImage: 'url(' + image + ')' }">
```

Comment:
instead of v-bind:style the background image, make id for the div instead or class